.ci/test-r-package-windows.ps1

```diff
@@ -176,16 +176,9 @@ Remove-Item "$env:RTOOLS_MINGW_BIN/cmake.exe" -Force -ErrorAction Ignore
 Write-Output "Done installing CMake"
 
 Write-Output "Installing dependencies"
-$packages = -join @(
-    "c('data.table', 'jsonlite', 'knitr', 'markdown', 'Matrix', 'processx', 'R6', 'RhpcBLASctl', 'testthat'), ",
-    "dependencies = c('Imports', 'Depends', 'LinkingTo')"
-)
-$params = -join @(
-    "options(install.packages.check.source = 'no'); ",
-    "install.packages($packages, repos = '$env:CRAN_MIRROR', type = 'binary', ",
-    "lib = '$env:R_LIB_PATH', Ncpus = parallel::detectCores())"
-)
-Invoke-R-Code-Redirect-Stderr $params ; Assert-Output $?
+Rscript.exe --vanilla ".ci/install-r-deps.R" --build --test ; Assert-Output $?
+# $command_args_patch = "commandArgs <- function(...){c('--build', '--test')};"
+# Invoke-R-Code-Redirect-Stderr "$command_args_patch source('install-r-deps.R')" ; Assert-Output $?
```

Comment:
```suggestion
```

This seems to work ok without the more complicated `Invoke-R-Code-Redirect-Stderr` pattern 😁